OpenSim/Actuators/TorqueActuator.h

```diff
@@ -114,18 +114,20 @@ class OSIMACTUATORS_API TorqueActuator : public ScalarActuator {
     bool getTorqueIsGlobal() const
     {   return get_torque_is_global(); }
 
-    /** Set the 'optimal_force' property. **/
+    /** %Set the 'optimal_force' property. **/
     void setOptimalForce(double optimalForce)
     {   set_optimal_force(optimalForce); }
     /** Get the current value of the 'optimal_force' property. **/
     double getOptimalForce() const override // Part of Actuator interface.
     {   return get_optimal_force(); }
 
-    /* Set the bodies to which this actuator applies torque. */
+    /** %Set the first body to which this actuator applies torque. */
```

Comment:
>Hmm..is there really a "first" and "second" body?

From the class description: "A TorqueActuatorr [sic… I'll fix the typo] applies equal and opposite torques on the two bodies (bodyA and B) that it connects."